gocourse09/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"fmt"
+
+	a "gocourse09/analyzer"
+	f "gocourse09/feeder"
+	z "gocourse09/zone"
```

Comment:
Краще не робити еліаси з однієї букви на імпорти:
```suggestion
	"gocourse09/analyzer"
	"gocourse09/feeder"
	"gocourse09/zone"
```

Це погіршує читабельність коду. Це конфліктує з назвами ресіверів, які зазвичай мають одну-дві букви. Це конфліктує з індексними змінними.